pkg/api/v1/summary_test.go

```diff
@@ -65,12 +65,12 @@ func TestListTabSummaries(t *testing.T) {
 				},
 			},
 			summaries: map[string]*summarypb.DashboardSummary{
-				"gs://default/summary/summary-acme": {},
+				"gs://default/summary/summary-testgroup1": {},
```

Comment:
nit: Just for clarity on this test case, think you can make this an empty map.

Reply:
+1, that would be clearer

Reply:
100% agree, done!